modules/admin_manual/pages/installation/deployment_recommendations/nfs.adoc

```diff
@@ -0,0 +1,146 @@
+* *Improved Security:* It {nfs-strong-security-architecture-url}[mandates a strong security architecture]. It does not require {rpc-statd-url}[rpc.statd] or {lockd-url}[lockd]. As a result, it only uses port 2049.
+* *Improved Reliability:* Uses TCP by default instead of UDP.
+* *Improved Performance:* It uses Multi-Component Messages, which reduce network traffic. It is capable of using a 32KB page size, compared to the default, 1024 bytes.
+* Use of {nfs-read-write-delegations-url}[Read/Write Delegations].
+
+== Tune the Mount Parameters
+
+=== _netdev
+
+Use this option to ensure that the NFS storage is available before ownCloud starts.
+It is essential when MySQL’s data files are located on the NFS storage, as MySQL could error if the connection to the storage is not ready when ownCloud starts.
+
+TIP: Alternatively, you can use {autofs-url}[autofs] to ensure that mounts are always available.
+
+=== soft
+
+We recommend this option to better handle storage failures.
+If `hard` is used, turning off the ownCloud server is likely to be problematic, and you need a boot and shutdown sequence in the servers.
+
+=== retrans
+
+Use this option to set the number of minor timeouts and retransmissions before major timeouts happen.
+Major timeouts either abort file operations or display the "server not responding" message.
+We recommend that you set it to ?.
+
+=== timeo
+
+Use this option, set in tenths of a second, to set the time before an initial retransmission after a timeout.
+
+According to {nfs-man-page-url}[the NFS man page], better overall performance may be achieved by increasing the timeout when mounting on a busy network, to a slow server, or through several routers or gateways. 
+We recommend that you set it to ?.
+
+=== noasync
+
+In clustered environments, we recommend setting the `noasync` option, to avoid filesystem I/O being handled asynchronously. 
+This has become the default on most distributions, but we recommend checking what this is set to in your ownCloud installations.
+
+=== noac
+
+The `noac` option should be used with caution. 
+Quoting the NFS man page:
+
+[quote, nfs man page]
+Using the noac option provides greater cache coherence among NFS clients accessing the same files, *but it extracts a significant performance penalty*. As such, judicious use of file locking is encouraged instead.
+
+== Tune the Read and Write Block Sizes
+
+The allowed block sizes are the packet chunk sizes that NFS uses when reading and writing data.
+The smaller the size, the greater the number of packets need to be sent to send or receive a file.
+Conversely, the larger the size, the fewer the number of packets need to be sent to send or receive a file.
+
+You can find this information by looking at the `/proc/mounts`, as in the example below.
+
+[source,console]
+----
+cat /proc/mounts | grep -i owncloud
+
+192.168.0.103:/data /mnt/owncloud nfs rw,vers=3,rsize=32768,wsize=32768,hard,proto=tcp,timeo=600,retrans=2,sec=sys,addr=192.168.0.103 0 0
```

Comment:
About the rsize and wsize  you have to mount first the storage withi a big number and test with:
(as example:)
```
#root@snoopy3:~# mount | grep /mnt/owncloud | egrep -o rsize=[0-9]*
rsize=65536
#root@snoopy3:~# mount | grep /mnt/owncloud | egrep -o wsize=[0-9]*
wsize=65536
```
The value that is shown there is the maximun transfer rate.

About the sixth field here is  0, it should be 2 because if the value is set to zero, the device or partition will be excluded from fsck check and if it is nonzero the fsck check will be run in the order in which the value is set. The mount partition is not a root partition